src/sinks/aws_kinesis_streams.rs

```diff
@@ -243,6 +244,8 @@ fn encode_event(
             .unwrap_or_default(),
     };
 
+    let data = Bytes::from(data);
```

Comment:
oh cool! they updated to use bytes! https://docs.rs/rusoto_kinesis/0.41.0/rusoto_kinesis/struct.PutRecordsRequestEntry.html#structfield.data woohoo!